1-resilient-messaging/docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
   app:
     image: simplycodedsoftware/php:8.2
     volumes:
-      - "$PWD:/data/app"
+      - ".:/data/app"
```

Comment:
@dgafka $PWD doesn't work on Windows so I replaced it with .
Can you check if this version work on Mac/Linux?

Reply:
Tak, będzie działać :)

Zmienisz też w drugiej częsci workshopu?

Reply:
Done